.obs/workflows.yml

```diff
@@ -3,6 +3,7 @@ workflow:
     - branch_package:
         source_project: OBS:Server:Unstable
         source_package: obs-server
+        target_project: OBS:Server:Unstable
```

Comment:
Currently we build here https://build.opensuse.org/project/subprojects/home:bs-team

Reply:
And we should keep doing this IMHO

Reply:
Ops, sure. Changed.